cfpb/consumerfinance.gov
# Expandables: Remove redundant visually hidden styles

The expandable label text is always hidden on mobile with `u-visually-hidden-on-mobile`. This class just applies the `u-visually-hidden` class at small screen sizes, so we can have either `u-visually-hidden-on-mobile` or `u-visually-hidden` and don't need both to be applied in some cases.

## Changes

- Expandables: Remove redundant visually hidden styles


## How to test this PR

1. Check a page that uses the hide_cue_label option and see that instead of `u-visually-hidden-on-mobile u-visually-hidden`, they only have `u-visually-hidden`.

These pages:
http://localhost:8000/data-research/prepaid-accounts/search-agreements/
e.g. http://localhost:8000/rules-policy/regulations/1003/4/
http://localhost:8000/consumer-tools/educator-tools/youth-financial-education/teach/activities/
